components/SubNav/SubNav.tsx

```diff
@@ -0,0 +1,38 @@
+import RCLink from "@/components/RCLink/RCLink"
+import styles from "@/styles/components/SubNav.module.scss"
+import { useRouter } from "next/router"
+
+/**
+ * The SubNav component renders the primary Research Catalog menu
+ * and passes the active prop to the correct link based on the route.
+ */
+const SubNav = () => {
+  const router = useRouter()
+
+  return (
+    <nav className={styles.subNav} aria-label="Research Catalog main menu">
+      <ul>
+        <li>
+          <RCLink href="/" active={router.pathname === "/"}>
```

Comment:
Similar comment regarding inline logic checks. There should be a more manageable and scalable way to have an `active` prop set.